samza-core/src/main/scala/org/apache/samza/coordinator/JobModelManager.scala

```diff
@@ -267,8 +266,6 @@ object JobModelManager extends Logging {
     * @return the input {@see SystemStreamPartition} of the samza job.
     */
   private def getInputStreamPartitions(config: Config, streamMetadataCache: StreamMetadataCache): Set[SystemStreamPartition] = {
```

Comment:
Could we add the following to the javadoc:

Computes the input system stream partitions of a samza job using the provided config. 
Adds side-inputs to inputSystemStreams, and expands all input topic regexes and populates them.

thanks